Formula/wcstools.rb

```diff
@@ -0,0 +1,16 @@
+class Wcstools < Formula
+  desc "Tools for using World Coordinate Systems (WCS) in astronomical images"
+  homepage "http://tdc-www.harvard.edu/wcstools/"
+  url "http://tdc-www.harvard.edu/software/wcstools/wcstools-3.9.5.tar.gz"
+  sha256 "b2f9be55fdec29f0c640028a9986771bfd6ab3d2f633953e4c7cc3b410e5fe9c"
+
+  def install
+    system "make", "-f", "Makefile.osx", "all"
+
+    prefix.install "bin"
+  end
+
+  test do
+    assert_match "IMHEAD", shell_output("#{bin}/imhead 2>&1", 1)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.